### DISCUSSION TITLE
ejabberd on Cloudron - One-Click deployment

### DISCUSSION THREAD

**Discussion by OrcVole** [processone/ejabberd#4095] [Questions]
We are nearly there with supporting ejabberd on Cloudron. Our developers just need a little bit of help from somebody who has experience of deploying ejabberd. 

If you would like to help, please introduce yourself on the thread here:

https://forum.cloudron.io/topic/2486/ejabberd-robust-scalable-and-extensible-realtime-server-using-xmpp-mqtt-and-sip

Cloudron is a platform to help people easily deploy web-facing applications. There is a live demo, if you would like to try it:

https://cloudron.io

**Comment by badlop** [comment]
Hi! Looking at that forum post, there are several people offering to help, and only one problem mentioned:
> I had a hard time so far to figure out the various overlaying ACL and access rules.

If there is a more detailed problem description, preferably with reproduction steps, I would be happy to give it a try.

**Reply by sando38** [reply]
Hello, 
I don't know how you guys deploy it in cloudron, but maybe this helm-chart may also help you:
https://github.com/sando38/helm-ejabberd

Like badlop, I am open to support as well.

**Reply by OrcVole** [reply]
@djxx has done a lot of the work necessary to support prosody on Cloudron. Perhaps that work will be of assistance to making ejabberd available with one click.

https://forum.cloudron.io/topic/10465/xmpp-server-prosody/17
